gdal-src/Cargo.toml

```diff
@@ -0,0 +1,314 @@
+[package]
+name = "gdal-src"
+version = "0.1.0"
```

Comment:
Will this be published separately?
If not, should it take the version from the workspace?

Reply:
Will do that.

Reply:
I just tried doing that, but there is currently no workspace wide version defined. It also seems like the `gdal` crate and the `gdal-sys` crate use different versions